config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```diff
@@ -318,13 +341,13 @@ spec:
                                 type: string
                                 maxLength: 128
                                 minLength: 1
-                                pattern: ^[0-9A-Za-z_.:/=+-@]+$
+                                pattern: ^[0-9A-Za-z_ .:/=+-@]+$
```

Comment:
@JoelSpeed @staebler can I slip the space in the Status in this PR as well? 

My thinking is:
* it's broadening functionality, not restricting
* values from .Spec are copied into .Status - values with spaces would fail

Reply:
Tested with a S3 resource:
key: `test-key: +_.:/=+-@abc-ABC-0123456789`
value `test-value: +_.:/=+-@abc-ABC-0123456789`

![Screenshot_20220406_145610](https://user-images.githubusercontent.com/4786168/161947931-663d7b9d-8d90-406a-b063-0cad6d7ad760.png)

Reply:
> values from .Spec are copied into .Status - values with spaces would fail

Reply:
I concur with leaving status as is.